docs/Daos.md

```diff
@@ -0,0 +1,252 @@
+# Working with DAOs
+
+Arc.js provides a class called [DAO](api/classes/DAO) that facilitates creating and working with DAOs.  Read on for more information about DAOs in DAOstack and the [DAO](api/classes/DAO) class.
+
+## About DAOs
+
+Every DAO in DAOstack has an architecture designed by Arc that enables it to use the reusable contracts provided by Arc.  Each DAO has an avatar, a controller, a token, a reputation system, and a set of schemes and global constraints.
+
+**The avatar** is a public-facing part of the DAO that handles the interaction of the DAO with the rest of the world, for example interacting with other DAOs or paying third party agents.  A DAO is always referenced by the address of its avatar.
```

Comment:
> A DAO is always referenced by the address of its avatar.

Or by its controller for the case it has a single controller.

Reply:
> Or by its controller for the case it has a single controller.

I don't understand this.  What is a "single controller"?  Do you mean "non-universal controller", and if so, isn't it true that only schemes can communicate with a controller (that used to be the case at least)?

Reply:
single controller  = non-universal controller .
by the end of the day scheme is register by its address ..so any registered address (account) can communicate with the controller.

Reply:
I am adding a note about this under **controller**